designsafe/apps/api/messages/models.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import unicode_literals
+import logging
+from django.db import models
+
+logger = logging.getLogger(__name__)
+TYPE_CHOICES = (
+    ('alert-warning', 'WARNING'),
+    ('alert-danger', 'DANGER'),
+    ('alert-info', 'INFO'),
+    ('alert-success', 'SUCCESS'),
+    )
+
+class PageAlert(models.Model):
+    alert_id = models.CharField(max_length=120, unique=True, primary_key=True)
```

Comment:
Why specifying a different field for the Id? By default Django uses `id` as a primary key.
Is this supposed to be filled by the admin user?

Reply:
Right, it's supposed to be filled out by an admin. Should the field be renamed to `name`?